### ISSUE TITLE
[BUG]: The btx-native coin has disappeared

### ISSUE THREAD

**Issue by Der-Garten-Pilz** [KomodoPlatform/atomicDEX-Desktop#2263]
**Describe the bug**
The btx-native & segwit coin has disappeared. 

**Expected behavior**
Would like to have it back :)

**Operating Environment(s):**
 - OS: Debian  Parrot os
 - OS version: Version 5.2 (Electro Ara) 64-bit
 -                     Kernel Linux 6.1.0-0.deb11.5-amd64 x86_64
 - AtomicDEX Version: 0.5.7.1-beta-ubuntu-appimage
 - Build branch: [e.g. master/dev] 

**Additional context**
   Only the btx.bep-20 is displayed.
   In Web-Dex all three are still present.

**Comment by cipig** [comment]
BTX has only one electrum server configured: https://github.com/KomodoPlatform/coins/blob/master/electrums/BTX
if that is down around the time the ADEX release is made, the coin is automatically removed from release
i can see it's back online, so will be re-added to next release
would be best if BTX community would run more electrum servers

**Comment by smk762** [comment]
If the BTX electrum server admin reaches out, we can invite him to a discord channel for server status alerts and troubleshooting.

**Comment by smk762** [comment]
Closing. This is not an app issue, it's an infrastructure issue on BTX side. Thanks for the report tho, sometimes we can reach out to the 3rd party to prompt them to update or fix infra.
I can also confirm that builds I did today include BTX, and as long as their electrums stay online, it will be in the 0.5.8 release.
![image](https://github.com/KomodoPlatform/atomicDEX-Desktop/assets/35845239/c3b1ebb4-5d6c-4d69-85b7-73190193b0bf)

Fwiw, they still only have one electrum listed, so it is at a higher risk of disappearing again than it would be if it have 2 or 3.
https://github.com/KomodoPlatform/coins/blob/master/electrums/BTX
